src/include/str_list.h

```diff
@@ -46,6 +49,8 @@ extern void get_str_vec(const std::string& str,
                          const char *delims,
 			 std::vector<std::string>& str_vec);
 
+std::vector<std::string> get_str_vec(const std::string& str,
+                                     const char *delims = ";,= \t");
```

Comment:
How do you feel about delims being a flat set and/or just taking any sequence of characters?

Reply:
i'm guessing the vast majority of cases will pass them a string literal, so i'd prefer not to copy them into something like a flat_set. but taking them as a string_view would be an easy option

Reply:
i decided to leave it as `const char*` here, because changing it to `string_view` would require changing the existing signatures too - and i don't think there's any real benefit to doing that